cli/js/testing.ts

```diff
@@ -307,6 +307,9 @@ function createFilterFn(
     if (filter) {
       if (filter instanceof RegExp) {
         passes = passes && filter.test(def.name);
+      } else if (/^\/.*\/$/.test(filter)) {
+        const filterAsRegex = new RegExp(filter.replace(/^\/|\/$/g, ""));
```

Comment:
`filter.slice(1, filter.length - 1)` is probably better than `filter.replace(/^\/|\/$/g, "")`

Reply:
Thanks @ry I guess I got really excited by REGEX 😅